website/content/en/blog/clusterlink-intro/index.md

```diff
@@ -0,0 +1,134 @@
+
+ClusterLink uses the Kubernetes API servers for its configuration store. The
+ **control plane** is responsible for watching for changes in relevant built-in
+ and custom resources and configuring the data plane Pod using [Envoy's
+ xDS](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/operations/dynamic_configuration)
+ protocol. The control plane is also responsible for managing local Kubernetes
+ services and endpoints corresponding to imported remote services. By using
+ standard Kubernetes services, ClusterLink integrates seamlessly with the Kubernetes
+ network model, and can work with any Kubernetes distribution, CNI and IP address
+ management scheme.
+
+The local service endpoints refer to **data plane** Pods, responsible for
+ workload-to-service secure tunnels to other clusters. The data plane uses
+ [HTTP CONNECT](https://en.wikipedia.org/wiki/HTTP_tunnel) with [mutual
+ TLS](https://en.wikipedia.org/wiki/Mutual_authentication#mTLS) for security.
+ The use of HTTPS over tcp/443 removes the need for VPNs and special firewall
+ configurations. Certificate based mTLS ensures guarantees in-transit data
+ encryption and limits connection only other fabric peers. In addition,
+ all data plane connections between clusters are explicitly approved by the
+ control plane and must pass independent egress and ingress access policies
+ before any workload data is carried across.
+
+### Use cases
+
+In addition to the typical multicluster networking use cases, such as
+ HA/DR, cloud bursting, and connecting microservices deployed across
+ geographically distributed clusters, ClusterLink can also provide
+ significant benefits which are not well served by other solutions.
+ Specifically, ClusterLink can address requirements of use cases where:
+
+- **clusters are aligned with organizational units** and sharing *internal*
+ microservices with other clusters and namespaces should be limited to those
+ belonging to the same unit, while also communicating with *exposed* services
+ belonging to other units.
+- services are owned by **different administrative domains** (e.g., different
+ development teams) and thus judicious sharing and more stringent access
+ controls across clusters are needed.
+- it is desirable to **increase scalability and limit information sharing** by
+ minimizing information exchanged between clusters. With ClusterLink, each
+ cluster manages its own naming and load balancing, requiring considerable
+ less cross cluster metadata for its communication.
+- there is a need for **separation of concerns** between network administrators
+ and application owners.
+
+## Getting started with ClusterLink
+
+To get started with ClusterLink, we invite you to explore the rest of the
+ documentation and familiarize yourself with its concepts and operation.
+ When ready, try out the [getting started tutorial]({{% ref "iperf" %}}).
+
+We would love to hear feedback and explore how we can make ClusterLink
```

Comment:
Maybe explicitly mention somewhere that ClusterLink is an open-source project with a very permissive license.